contribs/gmf/src/query/windowComponent.js

```diff
@@ -210,6 +214,12 @@ exports.Controller_ = function($element, $scope, ngeoQueryResult, ngeoMapQuerent
    */
   this.ngeoFeatureOverlayMgr_ = ngeoFeatureOverlayMgr;
 
+  /**
+   * @type {ngeo.datasource.DataSources}
+   * @private
+   */
+  this.ngeoDataSources_ = ngeoDataSources;
+
```

Comment:
Okay, I understand why we use the ngeoDataSources to get the ordered attributes and bind the data source id in the feature later to find it amond this list.

Would it make more sense to bind the data source to the window component instead?

Reply:
Not sure to understand what you expect, note that there can be features coming from multiple datasources at the same time in the query result / window components features collection.

Reply:
Oh... I hadn't figured that.  I looked at the component more and it would indeed be too complex to try to implement something there...  We have the "sources", a.k.a. the query result sources, that could be a point of entry that would be "unique per data source", but let's not do that for now.  It would be too complex.

Reply:
Note that we could point the datasource directly instead of datasource_id in feature properties.
No need to search for datasource after that.